src/python/pants/option/global_options.py

```diff
@@ -111,7 +111,7 @@ class ExecutionOptions:
     process_execution_use_local_cache: bool
     process_execution_local_enable_nailgun: bool
 
-    remote_store_server: List[str]
+    remote_store_addresses: list[str]
```

Comment:
I assume this PR is not where this option will be made singular?

Reply:
Yeah that can't happen till we finish the upcoming deprecation cycle unfortunately.